src/components/Attachment/Attachment.tsx

```diff
@@ -0,0 +1,445 @@
+    componentType: 'image',
+  });
+};
+
+export const renderCard = (props: InnerAttachmentUIComponentProps) => {
+  const { attachment: attachment, Card } = props;
+  if (!Card) return null;
+
+  if (attachment.actions && attachment.actions.length) {
+    return renderAttachmentWithinContainer({
+      attachment,
+      children: (
+        <div
+          className='str-chat__attachment'
+          key={`key-image-${attachment.id}`}
+        >
+          <Card {...attachment} key={`key-card-${attachment.id}`} />
+          {renderAttachmentActions(props)}
+        </div>
+      ),
+      componentType: 'card',
+    });
+  }
+
+  return renderAttachmentWithinContainer({
+    attachment,
+    children: <Card {...attachment} key={`key-card-${attachment.id}`} />,
+    componentType: 'card',
+  });
+};
+
+export const renderFile = (props: InnerAttachmentUIComponentProps) => {
+  const { attachment: attachment, File } = props;
+  if (!File || !attachment.asset_url) return null;
+
+  return renderAttachmentWithinContainer({
+    attachment,
+    children: (
+      <File attachment={attachment} key={`key-file-${attachment.id}`} />
+    ),
+    componentType: 'file',
+  });
+};
+
+export const renderAudio = (props: InnerAttachmentUIComponentProps) => {
+  const { attachment: attachment, Audio } = props;
+  if (!Audio) return null;
+
+  return renderAttachmentWithinContainer({
+    attachment,
+    children: (
+      <div className='str-chat__attachment' key={`key-video-${attachment.id}`}>
+        <Audio og={attachment} />
+      </div>
+    ),
+    componentType: 'audio',
+  });
+};
+
+export const renderMedia = (props: InnerAttachmentUIComponentProps) => {
+  const { attachment, Media } = props;
+  if (!Media) return null;
+
+  if (attachment.actions && attachment.actions.length) {
+    return renderAttachmentWithinContainer({
+      attachment,
+      children: (
+        <div
+          className='str-chat__attachment str-chat__attachment-media'
+          key={`key-video-${attachment.id}`}
+        >
+          <div className='str-chat__player-wrapper'>
+            <Media
+              className='react-player'
+              controls
+              height='100%'
+              url={attachment.asset_url}
+              width='100%'
+            />
+          </div>
+          {renderAttachmentActions(props)}
+        </div>
+      ),
+      componentType: 'media',
+    });
+  }
+
+  return renderAttachmentWithinContainer({
+    attachment,
+    children: (
+      <div
+        className='str-chat__player-wrapper'
+        key={`key-video-${attachment.id}`}
+      >
+        {' '}
+        <Media
+          className='react-player'
+          controls
+          height='100%'
+          url={attachment.asset_url}
+          width='100%'
+        />
+      </div>
+    ),
+    componentType: 'media',
+  });
+};
+
+/**
+ * Attachment - The message attachment
+ *
+ * @example ../../docs/Attachment.md
```

Comment:
for style guide to work more .md files need to be in same directory, so move in here and change path to ./Attachment.md